FEM/rf_bc_new.cpp

```diff
@@ -417,6 +417,12 @@ std::ios::pos_type CBoundaryCondition::Read(std::ifstream* bc_file,
 			in.clear();
 		}
 
+		if (line_string.find("VERTICAL_DISTRIBUTION") != std::string::npos)
```

Comment:
The VERTICAL_DISTRIBUTION key word will make the program to read in a curve from the RFD file. The values in between the data points will be interpolated linearly. This is necessary to assign boundary condition of GW temperature, that is vertically distributed due to geothermal gradient.